src/inc/eventtracebase.h

```diff
@@ -611,7 +611,7 @@ namespace ETW
 #else // FEATURE_EVENT_TRACE
     public:
         static VOID MethodJitting(MethodDesc *pMethodDesc, SString *namespaceOrClassName=NULL, SString *methodName=NULL, SString *methodSignature=NULL) {};
-        static VOID MethodJitted(MethodDesc *pMethodDesc, SString *namespaceOrClassName=NULL, SString *methodName=NULL, SString *methodSignature=NULL, SIZE_T pCode = 0, ReJITID rejitID = 0) {};
+        static VOID MethodJitted(MethodDesc *pMethodDesc, SString *namespaceOrClassName=NULL, SString *methodName=NULL, SString *methodSignature=NULL, SIZE_T pCode = 0, ReJITID rejitID = 0, BOOL bProfilerRejectedPrecompiledCode = FALSE, BOOL bReadyToRunRejectedPrecompiledCode = FALSE) {};
```

Comment:
@brianrob I don't understand this change. Why do we need to change the signature of `MethodJitted`?

Reply:
This is unrelated to the FEATURE_PERFTRACING.  It is related to #15801 which changed the signature of MethodJitted when FEATURE_EVENT_TRACE is defined, but not when it's not.

Reply:
@brianrob thanks!